docs/source/src_xyz_functions/chemcoord.xyz_functions.read_multiple_xyz.rst

```diff
@@ -0,0 +1,6 @@
+chemcoord.xyz\_functions.read\_multiple\_xyz
+================================
```

Comment:
can you extend the line till it extends to the whole title?

Otherwise it will throw a warning in Sphinx and in the future we aim to bring down the warnings to zero in sphinx.